map-generator/assets/maps/middleeast/info.json

```
@@ -0,0 +1,83 @@
+{
+  "name": "Middle East",
+ "nations": [
```

```suggestion
  "nations": [
```
prettier --write outputs this diff